reconcile/gitlab_housekeeping.py

```diff
@@ -371,7 +372,6 @@ def preprocess_merge_requests(
 
         item = {
             "mr": mr,
-            "labels": labels,
```

Comment:
follow up a fix as https://github.com/app-sre/qontract-reconcile/pull/3713

Reply:
@BumbleFeng Thank you!